### PR TITLE
Intly 9949 lower sso alert severity

### PR DESCRIPTION
**Pull request by mikenairn**
## Additional Information
JIRA: https://issues.redhat.com/browse/INTLY-9949

Related: https://github.com/integr8ly/keycloak-operator/pull/110

Lower selected sso alerts from critical to warning,  all these alerts don't meet the cssre critical alert criteria.

### Installation Verification
- Ensure the author of the PR has attached a log of the installation run from his branch to the jira or pr and check that it exited as expected.
- Verify the fresh installation is correct on cluster provided by PR author 
### Upgrade Verification
- After installation verification, notify the PR author to begin an upgrade on their cluster
- Ensure the developer of the PR has attached a log of the upgrade run from his branch to the jira or pr and check that it exited as expected. 
- If possible, look at the tasks that ran and see they match the PR
- Verify the upgrade is correct on cluster provided by PR author 

## Checklist

- [x] Updated [Changelog](https://github.com/integr8ly/installation/blob/master/CHANGELOG.md)
- [x] Tested Installation
- [x] Tested Uninstallation
- [x] Tested or Created follow on for Upgrade

Is an upgrade task required and are there additional steps needed to test this?
- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by mikenairn** [comment]
Verified here https://master.mnairn-69fa.open.redhat.com, install and upgrade.

**Comment by mikenairn** [comment]
/cc @philbrookes @pb82 @pmccarthy 

Would someone mind giving this a look over please.

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,11 +12,17 @@ Some of these changes may include:
 
 ## Unreleased
 
+* [INTLY-9949] - Update keycloak operator version 1.10.1. Lower the severity of sso alerts that do not meet the cssre critical alert criteria to warning.
+* [INTLY-9471] - Update Keycloak readiness/liveness probe
+* [INTLY-9948] - Lower the severity of threescale alerts that do not meet the cssre critical alert criteria to warning.
+* [INTLY-9907] - Lower all generic Kube* alerts to warning in 1.x
```
[the remark this answers is not in the document]
Comment (jessesarn) [reply CHANGELOG.md:18]: Looks like an oversight... not sure why, but totally agree we really should be taking advantage of the CHANGELOG.  No complaints about how you're approaching it here.

**Review by jessesarn — APPROVED** [review]
The cluster seems to have been cleaned back off since the install and find any location where I could get o the install/upgrade logs (could just be me being blind... been a crazy day/week).  However, the changes all look good and I did find the new image version on the cluster which tells me it obviously did get installed:
```
sha256:cb17bbcc7d835f2699b17404cca233c1e9a8cf6828dcb147daf170b4db3c08f1   registry.redhat.io/rh-sso-7/sso74-openshift-rhel8@sha256:cb17bbcc7d835f2699b17404cca233c1e9a8cf6828dcb147daf170b4db3c08f1
```
/lgtm
/approve

**Pull request merged**